infra/terraform/playbooks/citus.md

```diff
@@ -0,0 +1,94 @@
+# Citus
+
+Playbooks for interacting with the Citus database. Currently, we're running a
+single-node Citus setup on an EC2 instance.
```

Comment:
Do we want to rephrase this to make it clear that we're not actually running single-node Citus at the moment, but if we were using the Citus config _we would be_?

Reply:
```suggestion
This is a playbook for interacting with a self-hosted, single-node Citus cluster on EC2.
```